content/docs/traffic-management/transformations/templating-language.md

```diff
@@ -77,6 +77,7 @@ Apply transformation templates to add, set, or remode request or response header
 You can add request and response headers with static values. The following example adds the `my-header: static` request header and sets the `foo: bar` response header. 
 
 ```yaml
+
```

Comment:
does this still look normal in kgateway? i know it's needed for our docs framework, just not sure how it renders in hextra in kgw